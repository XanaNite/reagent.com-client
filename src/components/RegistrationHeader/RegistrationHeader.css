.regHeader h1{
    padding-left: 10px;
}

.regHeader {
    padding-left: 10px;
    padding-bottom: 2px;
    background-color: rgb(37, 23, 71);
    color: seashell;
    text-align: right;
}

.regHeader span{
    padding-left: 20px;
}

.regHeader a{
    color: seashell;
    text-decoration: none;
    margin-right: 5px;
}