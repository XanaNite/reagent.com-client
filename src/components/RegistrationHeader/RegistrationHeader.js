import React from 'react';
import {Link} from 'react-router-dom';
import './RegistrationHeader.css';

export default function RegistrationHeader(){
    return(
        <header role="banner" className='regHeader'>
            <nav role="navigation">
                <Link to='/login'>Log In</Link>
            </nav>
            <div className="signIn-title">
                <h1><Link to='/'>REagent.com</Link></h1>
                <span>for Professionals</span>
            </div>
        </header>
    );
}