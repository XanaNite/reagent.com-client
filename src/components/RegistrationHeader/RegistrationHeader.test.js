import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RegistrationHeader from './RegistrationHeader';

describe(`RegistrationHeader component`, () => {
    const props = {
        className: 'test-class-name',
        children: <p>test children</p>,
        'data-other': 'test-other-prop',
        agent: {
            first_name: "test",
            last_name: "test",
            title: "test",
            city: "test",
            state: "test"
        }
    };

    it('renders the RegistrationHeader given props', () => {
        const wrapper = shallow(<RegistrationHeader {...props} />)
        expect(toJson(wrapper)).toMatchSnapshot()
    });
})