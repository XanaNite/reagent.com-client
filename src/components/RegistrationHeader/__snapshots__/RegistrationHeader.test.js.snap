// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationHeader component renders the RegistrationHeader given props 1`] = `
<header
  className="regHeader"
  role="banner"
>
  <nav
    role="navigation"
  >
    <Link
      to="/login"
    >
      Log In
    </Link>
  </nav>
  <div
    className="signIn-title"
  >
    <h1>
      <Link
        to="/"
      >
        REagent.com
      </Link>
    </h1>
    <span>
      for Professionals
    </span>
  </div>
</header>
`;
